mantidproject/mantidimaging
# Basic UI for finding COR and tilt

Fixes #127

Basic first implementation. Good enough for preliminary review by instrument scientists.

Issues still present:
- Processing is single threaded
- Processing is done in GUI thread

To test:
- Open GUI
- Load `babylon5/DanNixon/PSI_Cylinder/Sample` (entire angle space with a step size of 40 should do)
- Select a region of interest that encloses one of the two large cylinders (that contain the several thinner cylinders), exceed the cylinder on the X axis but remain within the cylinder on the Y axis
- Open the Find COR and Tilt UI and click Set ROI from stack, the image preview should show the ROI you selected on the stack visualiser
- Enter 25 slices
- Click OK, progress should be visible in the terminal output
- You should see the COR per slice plotted in blue with a linear fit in red and a yellow line showing the sample tilt over the image
- This sample is not very tilted but you can check on the large image on the stack visualiser that the reported centre of rotation is correct (this is measured from the bottom of the ROI you selected)
